pkg/kf/commands/apps/push_test.go

```diff
@@ -29,20 +29,33 @@ import (
 	appsfake "github.com/google/kf/pkg/kf/apps/fake"
 	"github.com/google/kf/pkg/kf/commands/config"
 	"github.com/google/kf/pkg/kf/commands/utils"
+	servicebindings "github.com/google/kf/pkg/kf/service-bindings"
+	svbFake "github.com/google/kf/pkg/kf/service-bindings/fake"
 	"github.com/google/kf/pkg/kf/testutil"
+	"github.com/poy/service-catalog/pkg/apis/servicecatalog/v1beta1"
 )
 
+func dummyBindingInstance(appName, instanceName string) *v1beta1.ServiceBinding {
+	instance := v1beta1.ServiceBinding{}
+	instance.Name = fmt.Sprintf("kf-binding-%s-%s", appName, instanceName)
+
+	return &instance
+}
+
 func TestPushCommand(t *testing.T) {
 	t.Parallel()
 
 	for tn, tc := range map[string]struct {
-		args            []string
-		namespace       string
-		wantErr         error
-		pusherErr       error
-		srcImageBuilder SrcImageBuilderFunc
-		wantImagePrefix string
-		targetSpace     *v1alpha1.Space
+		args         []string
+		namespace    string
+		manifestFile string
+
+		wantErr                  error
+		pusherErr                error
+		srcImageBuilder          SrcImageBuilderFunc
+		serviceBindingClientFunc func(t *testing.T, f *svbFake.FakeClientInterface)
```

Comment:
nit: Can we rename `serviceBindingClientFunc` to `setup`? That's what we have elsewhere